src/scss/components/_listings.scss

```diff
@@ -0,0 +1,65 @@
+/**
+ * Listing
+ * ===================================
+ * Example – SERP List – Restaurant Information
+ *
+ *
+ */
+
+$listing-default-borderColor: $color-border;
+
+.c-listing {
```

Comment:
More a question for Damian for future work than a request for any changes on this PR, but would there ever be the possibility of being able to use the HTML sharing templates and pull this out into a full reusable HTML component?

Reply:
So, you mean we could have an `f-listings` npm module which would contain the SCSS and the templates @ashleynolan? I don't see why not, we just need to look a bit deeper into how we can compose templates together.

Reply:
Yeh – that's how I imagined it.

Maybe worth opening a JIRA, as it's not a massive priority right now, but would be cool to look into.  It's definitely a pattern that'll get re-used on other pages.